### PR TITLE
Bug 1946232: VM List high CPU usage

### PR DESCRIPTION
**Pull request by glekner**
New VM Table, Dropped the expensive conditions column
image

### CODE REVIEW

**Comment by glekner** [comment]
tests that are flaky for no apparent reason (passing locally) @gouyang @yaacov 

-  `tier1-a/advanced-wizard.spec.ts`
-  `tier1-a/clone.spec.ts`

**Comment by yaacov** [comment]
@glekner hi, note: we will need to add VMI phase, in this PR of a follow up

File: frontend/packages/kubevirt-plugin/src/components/vms/vm-page-new.tsx [hunk 1]
```diff
@@ -10,170 +10,179 @@ import {
...
+        {obj?.status?.printableStatus ? (
           <GenericStatus
-            title={printableStatus}
-            Icon={getVMStatusIcon(status, arePendingChanges)}
+            title={obj.status.printableStatus}
+            Icon={getVMStatusIcon(vmsStatus, arePendingChanges)}
           />
         ) : (
-          '-'
+          getStatusPhase(obj)
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/vms/vm-page-new.tsx:163]: +1

**Comment by yaacov** [comment]
/hold
tests are failing, need a deep looking into

**Comment by gouyang** [comment]
I have had a look at it locally, the selector `data-test="status-text"` is removed from vm status in the list view.
Please add it back and then let's see how it goes.

**Comment by gouyang** [comment]
try to fix the test failure in #10902

**Comment by gouyang** [comment]
@glekner Please disable this test `it('ID(CNV-4094): create template from PXE'` in this PR, the current Prow test environment has bug [2013160](https://bugzilla.redhat.com/show_bug.cgi?id=2013160), which is the status of an offline VM with HPP(don't check start on creation) is staying in "Provisioning" instead of "Stopped".

@yaacov Add Kobi for awareness.

**Pull request closed without merging**